src/components/cms/MyDevices/MapContainer.js

```diff
@@ -30,6 +30,9 @@ class MapContainer extends Component {
     google: PropTypes.object,
     invalidLocationDevices: PropTypes.number,
     adminTable: PropTypes.bool,
+    style: PropTypes.object,
+    locationFossasia: PropTypes.array,
+    tooltipRender: PropTypes.string,
```

Comment:
Keep only 1 prop instead of `devicesData` and `locationFossasia`, named `data`.
Make `tooltipRenderer` prop as a function and write rendering logic on the parent function.